VTEX - Intelligent Search Events API - Headless

```diff
@@ -0,0 +1,438 @@
+        "contact": {},
+        "version": "1.0"
+    },
+    "servers": [
+        {
+            "url": "https://sp.vtex.com/event-api/v1/{account}",
+            "description": "Server URL.",
+            "variables": {
+                "account": {
+                    "default": "account",
+                    "description": "Name of the VTEX account. Used as part of the URL."
+                }
+            }
+        }
+    ],
+    "paths": {
+        "/event": {
+            "post": {
+                "summary": "Save Events",
+                "description": "Creates search events to integrate with VTEX Intelligent Search using a headless implementation. \r\n\r\n >⚠️ **This API applies only to Headless scenarios**. It doesn't apply to stores using VTEX's storefront solution, natively integrated with Intelligent Search.",
+                "parameters": [],
```

Comment:
I get that you declared the `account` parameter in the server variables, but does it render normally as a path param if it is not on this array? I don't know the answer, but I guess it's worth keeping an eye out for this when it's time to merge.